Samples/StreamRecorder/StreamRecorderConverter/save_pclouds.py

```diff
@@ -160,9 +163,9 @@ def save_single_pcloud(shared_dict,
                                 {intrinsic_list[3]} \n")
 
                     # Create rgb and depth paths
-                    rgb_parts = Path(rgb_proj_path).parts[2:]
+                    rgb_parts = Path(rgb_proj_path).parts[3:]
                     rgb_tmp = Path(rgb_parts[1]) / Path(rgb_parts[2])
-                    depth_parts = Path(depth_proj_path).parts[2:]
+                    depth_parts = Path(depth_proj_path).parts[3:]
                     depth_tmp = Path(depth_parts[1]) / Path(depth_parts[2])
 
```

Comment:
the path of everyone is not the same, so the best solution is as follows:
```
rgb_parts = Path(rgb_proj_path).parts[2:]
rgb_tmp = Path(rgb_parts[-2]) / Path(rgb_parts[-1])
depth_parts = Path(depth_proj_path).parts[2:]
depth_tmp = Path(depth_parts[-2]) / Path(depth_parts[-1])
```
And **tsdf-integration.py** can be run correctly

Reply:
Yes, The filepath seems to be different for each OS.
I haven't tried it, but you are probably right.